exporter/chronicleexporter/README.md

```diff
@@ -48,8 +48,9 @@ currently supported log types are:
 - windows_event.system
 - sql_server
 
-
 If the `attributes["chronicle_log_type"]` field is present in the log, we will use its value in the payload instead of the automatic detection or the `log_type` in the config.
+If the `attributes["chronicle_namespace"]` field is present in the log, we will use its value in the payload.
+If there are nested fields in `attributes["chronicle_ingestion_label"]`, we will use the values in the payload.
```

Comment:
These should be broken out into a separate section from the "Log Type" section they're in currently, something like "Namespace & Ingestion Labels". Also state that if the attribute fields are present, they are used regardless of the value in the config.